apps/tlon-web/src/state/chat/chat.ts

```diff
@@ -544,6 +544,7 @@ function infiniteDMsUpdater(queryKey: QueryKey, data: WritDiff | WritResponse) {
               return;
             }
 
+            const newReplyId = unixToDa(memo.sent).toString();
```

Comment:
```suggestion
            const newReplyId = formatUd(unixToDa(memo.sent));
```

these keys have the dots